main.html

```diff
@@ -36,12 +36,14 @@
         <div class="title">    
             <h1>Federico Luna Vannini</h1>
             <h2 class="proffesion">Full Stack Dev</h2>
+            <div class="description">
+                <p>
+                I can help you build a website from the ground up, 
+                learning how to handle with the best practices at Microverse
+                </p>
+        </div>
         </div>
-        <p class="description">
-        I can help you build a website from the ground up, 
-        learning how to handle with the best practices at Microverse
-        </p>
-        <img src="./pictures/vector4.png" alt="line to separate information" class="separator">
+        <img src="./pictures/vector4.png" alt="line to separate information" class="separator head">
```

Comment:
- [ ] Your dividers aren't working, the issue is in the path, and the file you're trying to locate is with a different name, kindly fix that, also, to avoid these issues in the future, it's recommended that you use the `snake_case` while naming your resources, for example, in this case, it should be "vector_4.png", I recommend that you do this from now one 😁

![image](https://user-images.githubusercontent.com/20567503/177862557-e320a46e-a928-4808-9f7b-82c2d491c6bd.png)